lstchain/reco/r0_to_dl1.py

```diff
@@ -103,22 +103,12 @@ def get_dl1(
     """
 
     config = replace_config(standard_config, custom_config)
-    cleaning_parameters = config["tailcut"]
-    cleaning_parameters_for_tailcuts = cleaning_parameters.copy()
-    use_main_island = True
-    if "use_only_main_island" in cleaning_parameters.keys():
-        use_main_island = cleaning_parameters["use_only_main_island"]
-
-    # time constraint for image cleaning: require at least one neighbor
-    # within delta_time:
-    delta_time = None
-    if "delta_time" in cleaning_parameters:
-        delta_time = cleaning_parameters["delta_time"]
-
-    # we use pop because ctapipe won't recognize that keyword in tailcuts
-    cleaning_parameters_for_tailcuts.pop("delta_time")
-    cleaning_parameters_for_tailcuts.pop("use_only_main_island")
-    
+
+    # pop delta_time and use_main_island, so we can cleaning_parameters to tailcuts
+    cleaning_parameters = config["tailcut"].copy()
```

Comment:
The copy is still here @rlopezcoto

Reply:
ah ok, I missed that, thanks (test were passing, so it meant that changes were fine anyways)